test/load.js

```diff
@@ -0,0 +1,194 @@
+'use strict'
+
+const mkdirp = require('mkdirp').sync
+const rimraf = require('rimraf').sync
+const Persist = require('../')
+const test = require('tape')
+const path = require('path')
+const fs = require('fs')
+
+const folderName = 'data'
+const bucketName = 'bucket'
+const localPath = path.resolve(__dirname, folderName)
+const bucketPath = path.resolve(localPath, bucketName)
+
+class CustomPersister {
+  constructor (opts) {
+    this.values = {}
+    opts.initial.forEach((obj) => {
+      this.values[obj.name] = String(obj.value)
+    })
+  }
+
+  save (name, value) {
+    this.values[name] = value
```

Comment:
Yes-ish / No -ish. 

I can change the test to reflect it but save basically just does a Promise.all and returns it back for the caller to figure out what they want to do. https://github.com/littlstar/persist/blob/master/index.js#L46

Technically it would return successfully but it isn't very nice if the caller can't figure out what they did wrong. I'll fix the test just for standardization / example sake. :)